templates/Gemfile.erb

```diff
@@ -42,4 +42,8 @@ group :test do
   gem "webmock"
 end
 
+<% if SUSPENDERS_PATH.join("suspenders.gemspec").exist? %>
+gem "suspenders", path: "<%= SUSPENDERS_PATH %>", group: [:development, :test]
+<% else %>
```

Comment:
I wonder if there's any value in adding a comment that documents why we're doing this 🤔

```suggestion
<% if SUSPENDERS_PATH.join("suspenders.gemspec").exist? %>
# Use local install during develpoment
gem "suspenders", path: "<%= SUSPENDERS_PATH %>", group: [:development, :test]
<% else %>
```